### PR TITLE
fix: Fixes issue where CI was not executing all tests

### LINKED ISSUE #14333: [Bug]: CI does not catch Junit test failures
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When the PR is created in github multiple tests are run and if all of these tests run without failures then we can merge the changes to release branch. 

But the Junit test failures are not accounted for this. Even though there are failures in JUnit this is not considered and one can merge the PR to release branch. 
https://theappsmith.slack.com/archives/CPQNLFHTN/p1654535891273319

### PR DESCRIPTION
**Pull request by sidhantgoel**
## Description

> 

Fixes #14333

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by AnaghHegde — APPROVED** [review]

**Review by AnaghHegde — APPROVED** [review]

File: app/server/appsmith-interfaces/src/main/java/com/appsmith/external/plugins/PluginExecutor.java [hunk 1]
```diff
@@ -219,7 +219,6 @@ default Mono<TriggerResultDTO> trigger(TriggerRequestDTO request) {
      * @param actionConfiguration
      * @return modified actionConfiguration object after setting the two keys mentioned above in `formData`.
```
Comment (mohanarpit) [inline app/server/appsmith-interfaces/src/main/java/com/appsmith/external/plugins/PluginExecutor.java:220]: Apologies for the late comment. Can we please correct the function comment here? The code comment says that we are returning `ActionConfiguration` while the function is returning void.

**Pull request merged**